Hmm, the errors don't sound like anything related to my changes specifically:

```
##[error]1ES PT Error: Job 'ContentValidation' uses the release tasks ["AzureFunction@1","AzureFunction@1","AzureFunction@1"] but is not marked as a release job. Release tasks will soon be disallowed outside of release jobs. See [https://aka.***/1espt/custom-release-job](https://aka.%2A%2A%2A/1espt/custom-release-job)
```

I was expecting a runtime error if GStreamer wasn't available on the test system (missing DLL), but not sure what this is.

--- a/manifests/m/mtkennerly/madamiru/0.1.0/mtkennerly.madamiru.installer.yaml
+++ b/manifests/m/mtkennerly/madamiru/0.1.0/mtkennerly.madamiru.installer.yaml
@@ -28,15 +28,20 @@ FileExtensions:
 - wav
 - webm
 - webp
-Dependencies:
-  PackageDependencies:
-  - PackageIdentifier: gstreamerproject.gstreamer
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/mtkennerly/madamiru/releases/download/v0.1.0/madamiru-v0.1.0-win64.zip
   InstallerSha256: C7173669541BDE4318B7700C38A13D14EB31B9A94E7711AA92AA67A8140D51BE
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: gstreamerproject.gstreamer
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
 - Architecture: x86
   InstallerUrl: https://github.com/mtkennerly/madamiru/releases/download/v0.1.0/madamiru-v0.1.0-win32.zip
   InstallerSha256: C06C9675432C0509BEFE180AF7B29CEF90417012A0FE4E1E405C3AB647BA7198
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: gstreamerproject.gstreamer
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
 ManifestType: installer
 ManifestVersion: 1.9.0

